docs/authoring.md

```diff
@@ -0,0 +1,77 @@
+# Authoring
+
+Group Project XBlock v2 relies heavily on "nested XBlocks" feature, provided by edX Platform and XBlocks ecosystem.
+To be precise, Group Project is built from smaller XBlocks, each implementing some smaller subset of Group project 
+functionality: peer grading, submissions upload, team discussions, etc. Most of these features are internal to Group 
+Project XBlock v2 package, but some more advanced features reuse other XBlocks.
+
+Overall, the project structure is the following:
```

Comment:
I know you didn't write this @jbzdak but it might be worth clarifying that this is an example. It wasn't clear to me at first glance.